garden_ai/utils/misc.py

```diff
@@ -21,80 +16,6 @@
 issubtype = beartype.door.is_subhint
```

Comment:
`beartype` stuff can go too, I think this `issubtype` alias is the only reason it didn't get flagged as an unused import